pages/k8s/charm-kubernetes-worker.md

```diff
@@ -352,21 +352,307 @@ update the configuration for the workers:
-Juju users.
-
-### Pause
-
-Pausing the workload enables administrators to both drain and cordon
-a unit for maintenance.
-
-### Resume
-
-Resuming the workload will uncordon a paused unit. Workloads will automatically
-migrate unless otherwise directed via their application declaration.
-
+<!-- ACTIONS STARTS -->
+<!-- AUTOGENERATED TEXT - DO NOT EDIT -->
+
+The following actions are available. You can run an action with the following
+
+The following actions are available. You can run an action with the following ```bash juju run-action kubernetes-worker ACTION [parameters] [--wait] ```
+<div class="row">
+  <div class="col-2">
+    <h5>
+      cis-benchmark
+    </h5>
+  </div>
+  <div class="col-7">
+    <p>
+      Run the CIS Kubernetes Benchmark against snap-based components.
+    </p>
+  </div>
+</div>
+<div class="row">
+  <div class="col-2"></div>
+  <div class="col-7">
+    <p>
+      This action has the following parameters:
+    </p>
+    <table class="u-table-layout--auto">
+      <thead>
+        <tr>
+          <th>
+            parameter
+          </th>
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <pre>apply</pre>
+          </td>
+          <td>
+            Apply remediations to address benchmark failures. The default, 'none', will not attempt to fix any reported failures. Set to 'conservative' to resolve simple failures. Set to 'dangerous' to attempt to resolve all failures. Note: Applying any remediation may result in an unusable cluster. <strong>Default:</strong> none
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>config</pre>
+          </td>
+          <td>
+            Archive containing configuration files to use when running kube-bench. The default value is known to be compatible with snap components. When using a custom URL, append '#=' to verify the archive integrity when downloaded. <strong>Default:</strong> https://github.com/charmed-kubernetes/kube-bench-config/archive/cis-1.5.zip#sha1=cb8e78712ee5bfeab87d0ed7c139a83e88915530
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>release</pre>
+          </td>
+          <td>
+            Set the kube-bench release to run. If set to 'upstream', the action will compile and use a local kube-bench binary built from the master branch of the upstream repository: https://github.com/aquasecurity/kube-bench This value may also be set to an accessible archive containing a pre-built kube-bench binary, for example: https://github.com/aquasecurity/kube-bench/releases/download/v0.0.34/kube-bench_0.0.34_linux_amd64.tar.gz#sha256=f96d1fcfb84b18324f1299db074d41ef324a25be5b944e79619ad1a079fca077 <strong>Default:</strong> https://github.com/aquasecurity/kube-bench/releases/download/v0.2.3/kube-bench_0.2.3_linux_amd64.tar.gz#sha256=429a1db271689aafec009434ded1dea07a6685fee85a1deea638097c8512d548
```

Comment:
Maybe a notes table like for the config?  Annoying that both the description and default have that long URL.